### ISSUE TITLE
ME Security Terminal not put in Biometric Card

### ISSUE THREAD

**Issue by WispsFlame** [AppliedEnergistics/Applied-Energistics-2#3163]
**Description**
i want lock my AE system but my friend want to use so i try put the card into that but is not working@@

**Environment**
I play the modpack on Modern Skyblock 2

- Minecraft Version: 1.12.2
- AE2 Version: rv5-beta-1
- Forge Version: 1.12.2-14.23.0.2512

**Comment by GhostShot3** [comment]
I love this mod ...

**Comment by WispsFlame** [comment]
WT...

**Comment by GhostShot3** [comment]
Sorry random comments xD

**Comment by mindforger** [comment]
can you rephrase your issue a bit,. it looks like nobody understands the exact problem and what you have done exactly :)

**Comment by WispsFlame** [comment]
oh sorry my english so bad@@ so i mean i can't put the Biometric Card put into the ME Security Terminal

**Comment by yueh** [comment]
Fixed in dev,
